.taskcluster.yml

```diff
@@ -241,14 +241,14 @@ tasks:
           workerType: ci
           payload:
             maxRunTime: 3600
-            image: "mozilla/taskboot:0.2.5"
+            image: "mozilla/taskboot:0.2.7"
             env:
               WINDOWS_BIN: rust-code-analysis-cli.exe
             command:
                 - "/bin/sh"
                 - "-cx"
                 - "taskboot retrieve-artifact --output-path=. \
-                            --artifacts=$WINDOWS_BIN:public/$WINDOWS_BIN &&
+                            --artifacts=public/$WINDOWS_BIN &&
```

Comment:
I don't remember, did we change the `retrieve-artifact` command recently?

Anyway, this change seems unrelated to this PR, so please open a separate PR with it.

Reply:
Yep, we [did](https://github.com/mozilla/task-boot/commit/34c6274fd1af09b0bce163ec4649ab0deca245e4). 
I thought I could use the same PR to update everything :P
But you're right, I'm going to open a new PR

Reply:
Better to have each PR be a different set of logic changes (or each commit in a PR, but that's harder to handle as sometimes you'd have to rewrite history).

Reply:
I agree, but since they were both related to `TaskCluster`,I thought I could have used two different commits